File: nodejs/8 kyu/159.js
// https://www.codewars.com/kata/53d16bd82578b1fb5b00128c/train/javascript

// Passed

function grader(score) {
  if (score > 1 || score < 0.6) {
    return "F";
  } else if (score >= 0.9) {
    return "A";
  } else if (score >= 0.8) {
    return "B";
  } else if (score >= 0.7) {
    return "C";
  } else {
    return "D";
  }
}

const output = grader(0.9);
console.log(output);

File: python/7 kyu/159.py
# https://www.codewars.com/kata/58daa7617332e59593000006/train/python

# Passed

def find_longest(arr):
    longest_num = str()
    for num in arr:
        if len(str(num)) > len(str(longest_num)):
            longest_num = num
            
    return longest_num

output = find_longest([9000, 8, 800])
print(output)